shiptendance/app.js

```diff
@@ -0,0 +1,76 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+
+var routes = require('./routes/index');
+var users = require('./routes/users');
```

Comment:
Since you're not using any users, you can clean up this code and remove this from your app.js